### ISSUE TITLE
Unrealisticly high infiltration rates at soil thaw

### ISSUE THREAD

**Issue by mariuslam** [NGEET/fates#818]
Hi, 

When plotting QINFL (infiltration) in mm/day, (i just multiply by 60*60*24), I find super high rates when the top soil layer unfreezes. 

I compared Fates and Fates hydro in the most recent versions. It seems like this does not happen too often if all pft s are activated. But I run in only cold deciduous tree or only extratropical evergreen tree simulations.

Marius

![1x1_Yakutsk_era_QINFL](https://user-images.githubusercontent.com/44898967/145669987-8c00768f-07fa-4508-b995-0de7421a5e8b.png)
![1x1_Yakutsk_era_SNOWDP](https://user-images.githubusercontent.com/44898967/145669990-f58ad3a8-0d65-4c22-9712-dc515e0760c1.png)
![1x1_Yakutsk_era_TSL](https://user-images.githubusercontent.com/44898967/145669992-b6d1bf77-912a-413e-9e8c-33e40b886303.png)
![evergreen_QINFL](https://user-images.githubusercontent.com/44898967/145669996-9ea5aa1d-b175-489b-a9b3-b88a08d9a917.png)

**Comment by rgknox** [comment]
@mariuslam I'm trying to think of sources for all of this infiltrating water.  Presumably there wouldn't be enough precipitation and snow to melt to accomodate such large fluxes.

**Comment by olyson** [comment]
"(i just multiply by 606024)"
If you are converting from mm/s to mm/day, shouldn't this be 86400?

**Comment by mariuslam** [comment]
Yes , I meant 60min*60sec*24, so its the same as 86400. At the cold site , I encounter too high infiltration fluxes in several model setups and several model versions.
